CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- Fixed list jobs problems.
+- Updated list jobs unit test and system test.
```

Comment:
It doesn't hurt to have the extra info about the test being updated, however we usually leave out anything that may not impact end users.

Reply:
Thank you @zFernand0! I got it. 😊